functions/submission-created.js

```diff
@@ -0,0 +1,60 @@
+const sgMail = require('@sendgrid/mail');
+const { SENDGRID_API_KEY, ANGKLUNG_EMAIL } = process.env;
+
+// setup sendgrid
+sgMail.setApiKey(SENDGRID_API_KEY);
+
+exports.handler = async function (event, context, callback) {
+    const { name, email, message, formType } = JSON.parse(
+        event.body
+    ).payload.data;
+
+    console.log(`Received contact form submission from ${email}`);
+
+    // set up email msg. We are sending from our self to ourself.
+    const responseSubject = encodeURIComponent(
+        `Regarding your Angklung Ensemble enquiry, ${name}`
+    );
+    const msg = {
+        to: ANGKLUNG_EMAIL,
+        from: ANGKLUNG_EMAIL,
```

Comment:
Just to confirm, is this intentional to send from and to "ANGKLUNG_EMAIL"?
If that's the case, could you add a little comment there? Just to clarify doubts like the one that I have.